[ASM] - Rapid 7 Nexpose update (#24147)
demisto/content: Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/README.md

| Endpoint.OS | string | Endpoint operating system. | 
| CVE.ID | string | Common Vulnerabilities and Exposures IDs. | 

`nexpose-get-asset-tags`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | id | Asset ID. | Required |

Author: Resolved in 5751d2e.

| Endpoint.OS | string | Endpoint operating system. | 
| CVE.ID | string | Common Vulnerabilities and Exposures IDs. | 

`nexpose-get-asset-tags`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| asset_id | Asset ID. | Required |

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |

> | Nexpose.AssetTag.type | string | Type of asset tag |

Reviewer: ```suggestion
| Nexpose.AssetTag.type | string | Type of asset tag. |
```

Author: Resolved in c511d7d.

> | Nexpose.AssetTag.created_time | string | Timestamp with the tag was created |

Reviewer: ```suggestion
| Nexpose.AssetTag.created_time | string | Timestamp of when the tag was created. |
```

Author: Resolved in c511d7d.

> | Nexpose.AssetTag.name | string | The value of the tag |

Reviewer: ```suggestion
| Nexpose.AssetTag.name | string | The value of the tag. |
```

Author: Resolved in c511d7d.

> | Nexpose.AssetTag.risk_modifier | string | The risk modifier value associated with criticality tag type |

Reviewer: ```suggestion
| Nexpose.AssetTag.risk_modifier | string | The risk modifier value associated with criticality tag type. |
```

Author: Resolved in c511d7d.

> #### Command example

Reviewer 2: Please update the Human Readable Output too

Author: Resolved in 7e30033.
